ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productlist/v1/productlist/README.md

```diff
@@ -0,0 +1,67 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Product List (v1)
+====
+Product List component written in HTL, allowing to display product listings on a category page.
+
+## Features
```

Comment:
I think we need to clarify more things here:
* It is a fully server-side rendered component
* It retrieves product data via GraphQL
* It is supposed to be used on a generic category page only
* It requires a category id to be passed as a sling selector to identify the category to display

Reply:
Done